projectq/backends/_circuits/_to_latex.py

```diff
@@ -234,7 +245,7 @@ def __init__(self, settings, num_lines):
         self.op_count = [0] * num_lines
         self.is_quantum = [settings['lines']['init_quantum']] * num_lines
 
-    def to_tikz(self, line, circuit, end=None):
+    def to_tikz(self, line, circuit, end=None, one_gate_at_a_time = False):
```

Comment:
Please document the new function argument inside the docstring.

Reply:
new PR addresses this comment